src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -462,7 +462,16 @@
 examples:
   - name: Get a security setting.
     text: >
-        az security setting show -n "MCAS"
+        az security setting show --setting-name "MCAS"
+"""
+
+helps['security setting update'] = """
+type: command
+short-summary: Updates a security setting.
+examples:
+  - name: Update a security setting.
+    text: >
+        az security setting update --setting-name "Sentinel" --enabled true
```

Comment:
I recommend to keep using `--name` and `-n` instead of `--setting-name` in this command. You can set `options_list=['--name', '-n']` for `setting_name` argument.

```suggestion
        az security setting show -n "MCAS"
"""

helps['security setting update'] = """
type: command
short-summary: Updates a security setting.
examples:
  - name: Update a security setting.
    text: >
        az security setting update --name "Sentinel" --enabled true
```